rfcs/text/0000-metadata-integration-test.md

```diff
@@ -20,19 +20,20 @@ This document details a integration test design for metadata schema changes in t
 
 Currently breakages to DCP systems caused by metadata schema changes are detected at runtime upon data (bundle) upload. This forces data wranglers, DCP operators, and data consumers to react to unpredictable runtime failures rather than detecting problems before the code is merged and deployed.
 
-To alleviate this issue, the DCP development team can use an metadata schema integration test to test whether it is safe to release data downstream of the Data Storage Service (DSS), past which point applications start to depend on the schema. The benefits of this approach are that it will:
+To alleviate this issue, the DCP development team can use a metadata schema integration test to test whether it is safe to release data downstream of the Data Storage Service (DSS), past which point applications start to depend on the schema. The benefits of this approach are that it will:
 
-1. Make the development environment more stable
-1. Make updates to software and schema smaller and more frequent
-1. Make failures due to metadata schema changes more readily observable
-1. Increase availability of DCP systems downstream of DSS
+1. Make the development environment more stable.
+1. Make updates to software and schema smaller and more frequent.
+1. Make failures due to metadata schema changes more readily observable.
+1. Increase availability of DCP systems downstream of DSS.
 
 ## Detailed Design
 
 In this design we will use the following terms:
  
- * "schema integration test system" - an automated CI/CD pipeline used to check that downstream systems can handle bundles with a new schema
- * "new schema label" - anything that indicates that the schema has changes that have not yet been tested against downstream systems: potentially a JSON field, git tag, or git branch
+ * "schema integration test system" - an automated CI/CD pipeline used to check that downstream systems can handle bundles with a new schema.
+ * "new schema label" - anything that indicates that the schema has changes that have not yet been tested against downstream systems: potentially a JSON field, git tag, git branch, or comparison to tested version list.
```

Comment:
nit: I don't usually put periods at the end of sentences if they are not full sentences.